### PR TITLE
Quickstart: add SEV annotations information

### PR DESCRIPTION
**Pull request by fitzthum**
SEV configuration no longer requires updating the Kata config file directly. Instead, configuration can be done via pod annotation.

CC: @ryansavino @jimcadden

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -437,40 +437,30 @@ sudo ./sevctl/target/debug/sevctl export --full /opt/sev/cert_chain.cert
...
-
-The `simple-kbs` is a prototype implementation under development and is not intended for production use at this time.
-
-For the SEV encrypted image use case, it is required to host the key used to encrypt the container image from the `simple-kbs`.
+By default, the `kata-qemu-sev` runtime class uses pre-attestation with the
+`online-sev-kbc` and [simple-kbs](https://github.com/confidential-containers/simple-kbs) to attest the guest and provision secrets.
+`simple-kbs` is basic prototype key broker that can validate a guest measurement according to a policy and conditionally release secrets.
+To use encrypted images, signed images, or authenticated registries with SEV, you should setup `simple-kbs`.
+If you simply want to run an unencrypted container image, you can disable pre-attestation by adding the following annotation
+`io.katacontainers.config.pre_attestation.enabled: "true"` to your pod.
```
Comment (ryansavino) [inline quickstart.md:445]: should this be false?

Comment (fitzthum) [reply quickstart.md:445]: whoops

**Review by ryansavino — APPROVED** [review]
LGTM

**Comment by fitzthum** [comment]
@confidential-containers/documentation-maintainers can someone take a look at this

**Comment by fitzthum** [comment]
I'm going to add this stuff as part of another PR that is coming soon.

**Pull request closed without merging**